content/docs/concepts/design-principles.mdx

```diff
@@ -1,209 +1,207 @@
 subdirectories in the kernel source tree to identify (broad) components. [We
 used `cscope` to extract all function calls from the sources of all kernel
-components](https://github.com/unikraft/eurosys21-artifacts/tree/master/experiments/fig_01_linux-deps),
-and then for each call checked to see if the function is defined in the same
-component or a different one; in the latter case, we recorded a dependency.  We
-plot the dependency graph in Figure 1, the annotations on the edges show the
-number of dependencies between nodes.  This dense graph makes it obvious that
-removing or replacing any single component in the Linux kernel requires
-understanding and fixing all the dependencies of other components, a daunting
-task.
+components][linux-deps-exp]. For each function call, we checked if the function
+definition existed in the same component or a different one. In the latter case,
+we recorded a dependency. We plot the dependency graph in Figure 1 where the
+annotations on the edges show the number of dependencies between nodes. This
+dense graph makes it obvious that removing or replacing any single component in
+the Linux kernel requires understanding and fixing all the dependencies of other
+components, a daunting task indeed.
 
 <Image
   border
-  ratio={3/2}
+  ratio={3 / 2}
   maxW={'2xl'}
-  src="https://raw.githubusercontent.com/unikraft/eurosys21-artifacts/master/plots/fig_01_linux-deps.svg"
-  title="Figure 1"
-  description="Linux kernel components have strong inter-dependencies, making it
-               difficult to remove or replace them."
+  src='https://raw.githubusercontent.com/unikraft/eurosys21-artifacts/master/plots/fig_01_linux-deps.svg'
+  title='Figure 1'
+  description='Linux kernel components have strong inter-dependencies, making it
+               difficult to remove or replace them.'
 />
 
-While full modularization is difficult, modularizing certain parts of a
-monolithic kernel has been done succesfully by Rump. There, the NetBSD kernel
-was split into base layers (which must be used by all kernels), functions
-provided by the host (scheduling, memory allocation,etc) and so-called factions
-that can be run on their own (e.g.  network or filesystem support). Rump goes
-some way towards achieving our goals, however there are still many dependencies
-left which require that all kernels have the base and hypercall layers.
-Additionally, the dependencies on the host are limiting in the context of a VM,
-which is our target deployment.
-
+While full modularization is difficult, rumpkernel has modularized certain parts
+of a monolithic kernel. They split the NetBSD kernel into base layers (used by
+all kernels), functions provided by the host (scheduling, memory allocation,
+etc.), and independently executable "factions" (e.g., network or filesystem
+support). rumpkernel goes some way towards achieving our goals. Still, many
+dependencies remain that require the kernel to have the base and hypercall
```

Comment:
"Still, many
dependencies remain that require the kernel to have the base and hypercall
layers. In the context of a VM, which is our deployment target, this limits the
extent of modularity that we can achieve."

Please check if this still carries the meaning the author initially intended.